custom_components/sonnenbatterie/config_flow.py

```diff
@@ -4,125 +4,188 @@
 # pylint: enable=no-name-in-module
 import traceback
 
-# import logging
-# import voluptuous as vol
-from homeassistant import config_entries, core
+from homeassistant import config_entries
+from homeassistant.const import (
+    CONF_IP_ADDRESS,
+    CONF_PASSWORD,
+    CONF_SCAN_INTERVAL,
+    CONF_USERNAME,
+)
 from homeassistant.core import callback
 from homeassistant.helpers import config_validation as cv
 
 # pylint: disable=unused-wildcard-import
-from .const import *  #
+from .const import *
 
 # pylint: enable=unused-wildcard-import
 import voluptuous as vol
-from homeassistant.const import (
-    CONF_PASSWORD,
-    CONF_USERNAME,
-    CONF_IP_ADDRESS,
-    CONF_SCAN_INTERVAL,
-)
 
 
 class SonnenbatterieFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
-    def __init__(self):
-        """Initialize."""
-        self.data_schema = CONFIG_SCHEMA_A
+
+    CONFIG_SCHEMA_USER = vol.Schema(
+        {
+            vol.Required(CONF_IP_ADDRESS, default="127.0.0.1"): str,
```

Comment:
I am not sure if we should 127.0.0.1 as the default... i think this value would never work unless you run ha on your sonnenbatterie :)